tests/SampleApps/python/python2-flask-app/requirements.txt

```diff
@@ -1 +1,2 @@
 Flask==0.9
+Werkzeug==0.9
```

Comment:
Keeping it the same as in `master` branch: https://github.com/microsoft/Oryx/commit/ce3e224fccf49aaa125ff7ecbeac89e0d52d0fc1#diff-03b69b14e4a6284268c18375b99e9eae